splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -76,6 +78,18 @@ public class SplunkIntegration extends EndpointRouteBuilder {
     // The return type
     public static final String RETURN_TYPE = "com.redhat.console.notifications.history";
 
+    class SplunkHttpHeaderStrategy extends HttpHeaderFilterStrategy {
+        @Override
+        protected void initialize() {
+            setLowerCase(true);
+            setFilterOnMatch​(false); // reverse filtering to only accept selected
+
+            getInFilter().clear();
```

Comment:
just a question about `getInFilter().clear()`, as I am not sure and I did not understand it from the docs... how does this influence the resulting set of headers?

Reply:
This is a filter of headers. In Exchange there is `in` message and `out` message. This resets filter for the `in` message headers. I guess, the in headers are what the http component gets and the out ones are what it generates. At least that's my understanding. 

I can double check it, but I've already seen that it passed `Camel*` internal headers if this was not reset. And that was also due to the reverse of the matching (the `Camel*` headers are filtered by default).

Reply:
Okay, so since there is no `super.initialize()` call to use whatever the parent class sets, the clear is redundant. I'll keep it there just to be sure for future.